bazel/internal/scan/scan.bzl

```diff
@@ -165,8 +165,12 @@ def scan_impl(ctx):
     xcmappingmodels = sets.to_list(xcmappingmodels_set)
     test_targets = sets.to_list(test_targets_set)
 
+    indexstores_config = [file.path for file in indexstores]
+    if ctx.attr.global_indexstore:
```

Comment:
I wonder if this should be automatic based on the value of `use_global_index_store`?